src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -46,6 +46,22 @@
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "neighbour";
 
+    public static final String USER_TIP = "Try adding a student using: "
```

Comment:
I think "Try ..." sounds ok for new users, but for long time users it may sound a bit strange. But not major issue, i think can just leave it like this first.